stdlib/public/Cxx/CxxDictionary.swift

```diff
@@ -72,4 +75,22 @@ extension CxxDictionary {
       }
     }
   }
+  
+  func filter(
+    _ isIncluded: (_ key: Key, _ value: Value) throws -> Bool
+) rethrows -> [Key: Value] {
+    var filteredDictionary: [Key: Value] = [:]
+    var iterator = __beginUnsafe(Key) 
```

Comment:
This won't compile since `__beginUnsafe` takes no parameters. Please try building this on your machine to see the compiler error, feel free to ask me questions if you run into issues when building Swift.